src/annotator/components/adder-toolbar.js

```diff
@@ -16,14 +16,18 @@ function ToolbarButton({ icon, label, onClick, shortcut }) {
       onClick={onClick}
       title={title}
     >
-      <SvgIcon name={icon} />
+      {icon && <SvgIcon name={icon} />}
+      {typeof badgeCount === 'number' && (
```

Comment:
Given that `propTypes` defines this prop as being a  `number`, can you help me understand the need for the `typeof` check here? Sorry if I missed it in past discussion.

Reply:
The prop is optional, so it may be `undefined`. I could have set a default value of 0 and used `{badgeCount > 0 && ...}`.